python/train.py

```diff
@@ -182,8 +182,8 @@ def main( args ):
                 err = err / vn / pars['Num_iter_per_show']
                 cls = cls / vn / pars['Num_iter_per_show']
             else:
-                err = err / num_mask_voxels / pars['Num_iter_per_show']
```

Comment:
Sure. The num_mask_voxels variable accumulates over the number of iterations anyway, so we don't need to further normalize it by the number of iterations. 

For example, if there were 10 rounds of 10 mask voxels, the num_mask_voxels value would be 100, and diving by 10 at the end obscures the resulting error value.